src/core/providers/ogr/qgsogrprovider.cpp

```diff
@@ -512,6 +513,17 @@ QgsOgrProvider::QgsOgrProvider( QString const &uri, const ProviderOptions &optio
                           mSubsetString,
                           mOgrGeometryTypeFilter );
 
+  if ( mFilePath.contains( QLatin1String( "authcfg" ) ) )
+  {
+    QRegularExpression authcfgRe( " authcfg='([^']+)'" );
```

Comment:
Is the space at the start deliberate?

Reply:
yes. Taken from another occurence in that file